webapp/shadow-player/src/websocket.ts

```diff
@@ -15,8 +20,13 @@ export class ServerWebSocket {
       const reader = new FileReader();
       reader.onload = () => {
         const arrayBuffer = reader.result as ArrayBuffer;
-        const serverResponse = parseServerMessage(arrayBuffer);
-        callback(serverResponse);
+        try {
```

Comment:
prepare streamer player for future features.

Reply:
I don’t understand how it’s "preparing". Typically I would suggest to do the changes when they are actually necessary, but let me know.

Reply:
You are right, I'll remove this part as well.